services/iota_wrappers.go

```diff
@@ -139,11 +142,18 @@ func init() {
 
 	provider := "http://" + host_ip + ":14265"
 
-	api = giota.NewAPI(provider, nil)
+	// create a new API instance
+	api, err = giota.ComposeAPI(giota.HttpClientSettings{
+		URI:          provider,
+		LocalPowFunc: bestPow,
+	})
+	if err != nil {
+		panic(err)
+	}
 
 	seed = "OYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRL"
 
-	powName, bestPow = giota.GetBestPoW()
+	powName, bestPow = pow.GetBestPoW()
```

Comment:
This is not an error that you made, but we could fix it in this PR anyway.  This line should probably be moved up above line 145, before the api is instantiated.

Reply:
Good catch!